tests/e2e/cases/09_fpm_run.sh

```
@@ -0,0 +1,37 @@
+    sleep 0.1
+done
+
+if ! nc -z 127.0.0.1 9000 2>/dev/null; then
+    echo "--- stdout ---"; cat /tmp/fpm.stdout || true
+    echo "--- stderr ---"; cat /tmp/fpm.stderr || true
+    fail "TCP :9000 not listening"
+fi
+[[ -S "$FPM_SOCK" ]] || fail "unix socket $FPM_SOCK not created"
+
+WORKER_COUNT=$(pgrep -P "$FPM_PID" 2>/dev/null | wc -l || echo 0)
+[[ "$WORKER_COUNT" -ge 2 ]] && ok "$WORKER_COUNT workers spawned" \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)